chore: Update staking farm related copy

MaximusHaximus opened the pull request:
1. Alphabetizes the JSON files now that we have automatic tooling added to handle this whenever they are edited
2. Updates copy for token farm validators

MaximusHaximus commented:
@corwinharrell Would like your input on the new tooltip text and the new balance of tokens earned via the staking farm.

```
                "info": "This is a 'staking farm'. You will receive tokens other than NEAR as rewards while staking on it.",
                "title": "Un-claimed farm tokens earned"
```

corwinharrell commented:
@MaximusHaximus Could you add screenshots of the affected UI in context? Just helps to review things like this without going through setup locally. Would love to get into a habit of this any time there are changes that affect the front-end/UI

MaximusHaximus commented:
@corwinharrell You don't need to set anything up locally to see how changes look in context, since we automatically deploy previews for every PR with both testnet and mainnet configurations.  

- You can see link for the mainnet staging preview under the 'checks' area of the PR at the bottom of this screen -- click 'Details' next to the row labelled `netlify/near-wallet-staging/deploy-preview — Deploy Preview ready!`.
- You can see the testnet preview build by using the link in the comment that Render added to this PR as the first response comment (https://github.com/near/near-wallet/pull/2388#issuecomment-1017004487)

Here's a picture of the screen with the new terms being rendered for your convenience, but it would probably be a good idea for you to get in the habit of using the preview builds in general, since screenshots are pretty limited (especially for changes that can impact multiple screens)
![image](https://user-images.githubusercontent.com/29758232/150430668-8a24fb48-5bb8-4ae9-af73-125c27d3d1f4.png)

corwinharrell commented:
Sorry for the miscommunication @MaximusHaximus. I'm totally comfortable using the deploy links. My comment about adding an image was purely an aside and for convenience sake for future PRs. Didn't mean to suggest it was a blocker. Had to step away for a bit this afternoon but looking at this now.

corwinharrell commented on packages/frontend/src/translations/en.global.json:
```diff
@@ -1226,8 +1226,8 @@
-                "info": "This is project validator. You’ll get the farming reward while staking on it.",
-                "title": "Total token earned"
+                "info": "This is a 'staking farm'. You will receive tokens other than NEAR as rewards while staking on it.",
+                "title": "Total tokens earned"
```
How would we feel about:

"Additional tokens earned"

corwinharrell commented on packages/frontend/src/translations/en.global.json:
```diff
@@ -1226,8 +1226,8 @@
                 "button": "Claim",
-                "info": "This is project validator. You’ll get the farming reward while staking on it.",
-                "title": "Total token earned"
+                "info": "This is a 'staking farm'. You will receive tokens other than NEAR as rewards while staking on it.",
```
How about:

"This pool awards delegator stake with tokens other than NEAR."

corwinharrell commented:
<img width="563" alt="Screen Shot 2022-01-20 at 6 47 43 PM" src="https://user-images.githubusercontent.com/5934188/150440165-5152122b-b58c-4aab-a715-60ca4cd1cad0.png">

Until we have a new UI, I think it is actually clearest to group the rewards under a single section and label. This way, it simply becomes a list of rewards earned, and the icon and symbol denote the token, without the need for separate labels and sections.

I think the only remaining necessary change then would be to change the tooltip icon to read something like:

"Rewards earned but not withdrawn. NEAR token rewards are compounding and are automatically re-staked."

corwinharrell commented:
@MaximusHaximus See this comment: https://github.com/near/near-wallet/issues/2383#issuecomment-1018028825 since I think it has a direct impact on the changes proposed in this PR.

MaximusHaximus commented:
@corwinharrell I modified the UI to look like the picture you've created, and modified the translation terms accordingly -- thanks for the feedback/direction!

MaximusHaximus commented:
@Patrick1904 I'm pretty sure my usage of dynamic props for the container here isn't what you'd want given I haven't seen this pattern in other code -- I'm shipping with that setup in the interest of time but am keen to have a chat with you when you're back about the right way to handle the border formatting in the list 'the right way, and to understand the benefits of avoiding dynamic props usage for our style components if that's what we're generally trying to do as a team. :)
